### ISSUE TITLE
Provide step-by-step instructions to manually deploy and setup the reference Diamond implementations

### ISSUE THREAD

**Issue by Ungolim** [mudgen/diamond#2]
I don't want to rely on hardhat-deploy built-in diamond support boilerplates and default steps to deploy and setup the reference implementations; right now, the most accurate steps can be evinced from the test suites (https://github.com/mudgen/diamond-1/blob/master/test/diamondTest.js) and the migrations (https://github.com/mudgen/diamond-1/blob/master/migrations/2_diamond.js)

**Comment by mudgen** [comment]
Yes, I agree with you.  Would love it if someone could help with this.

**Comment by Sufiyan-dev** [comment]
hey @mudgen i would love to help you out

**Comment by mudgen** [comment]
@Sufiyan-dev 

> hey @mudgen i would love to help you out

Great!
